tasks/prime.py

```diff
@@ -272,7 +274,80 @@ def on_start(self):
             name="/internal/service_members/{serviceMemberId}/backup_contacts",
             data=json.dumps(payload),
             headers={"content-type": "application/json"},
-            **self.user.cert_kwargs,
+        )
+
+        # Setup customer order
+        overrides = {
+            "service_member_id": service_member_id,
+            "issue_date": datetime.now().strftime("%Y-%m-%d"),
+            "report_by_date": datetime.now().strftime("%Y-%m-%d"),
+            "orders_type": "PERMANENT_CHANGE_OF_STATION",
+            "orders_type_detail": "HHG_PERMITTED",
+            "has_dependents": False,
+            "spouse_has_pro_gear": False,
+            "new_duty_station_id": stations[1]["id"],
+            "orders_number": None,
+            "tac": None,
+            "sac": None,
+            "department_indicator": None,
+        }
+        payload = self.fake_request("/orders", "post", INTERNAL_API_KEY, overrides, True)
```

Comment:
To Felipe's point I would say at least after every request if we could check that it didn't error before using the result that would be helpful to know.